### PR TITLE
[DOC] Add feature toggle for admin page

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What is this feature?**

Adds the `featureToggleAdminPage` feature toggle to enable the Feature toggles admin page in Grafana. 

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/administration/feature-toggles/index.md [hunk 1]
```diff
@@ -51,5 +51,7 @@ To edit a feature toggle, follow these steps:
 1. Click **Save changes** and wait for your Grafana instance to restart with the updated feature toggles.
 
 {{% admonition type="note" %}}
+If you don't have the feature toggle management page, enable the `featureToggleAdminPage` feature toggle.
+
 Editing feature toggles with the feature toggle management page is available now in all tiers of [Grafana Cloud](/docs/grafana-cloud/).
 {{% /admonition %}}
```
Comment (jdbaldry) [inline docs/sources/administration/feature-toggles/index.md:57]: nit: while we are here:

```suggestion
{{< admonition type="note" >}}
If you don't have the feature toggle management page, enable the `featureToggleAdminPage` feature toggle.

Editing feature toggles with the feature toggle management page is available now in all tiers of [Grafana Cloud](/docs/grafana-cloud/).
{{< /admonition >}}
```

**Review by jdbaldry — APPROVED** [review]
LGTM

File: docs/sources/administration/feature-toggles/index.md (hunk 1 above)
Comment (knylander-grafana) [reply docs/sources/administration/feature-toggles/index.md:57]: Thank you.

**Comment by diegommm** [comment]
Hi! Greetings from the Release Guild. Just letting you know I'm updating your PR's milestone to [11.2.x](https://github.com/grafana/grafana/milestone/529). If you need to see your changes in a previous version, please consider adding a backport label.

Thank you!

**Pull request merged**